packages/dropdown/src/Default/Default.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import { DropdownDefaultProps, DropDownItemProps } from './Default.types';
+import * as styleMod from './styles.module.scss';
+import { createLocalProps } from '@owlui/utils';
+import { Dropdown, ThemeProvider } from 'react-bootstrap';
+
+export const Component = (props: DropdownDefaultProps) => {
+  const baseClass = 'dropdown';
+  const prefix = props.prefix || '';
+  const { dropdown } = props;
+  const [show, setShow] = React.useState(false);
+  const [selectedItem, setSelectedItem] = React.useState<number | null>(null);
+
+  const locals = createLocalProps(
+    props,
+    {
+      module: styleMod,
+      classes: {
+        base: baseClass,
+        prefix: prefix,
+        optionals: [
+          {
+            fields: ['theme'],
+            value: 'theme',
+          },
+          {
+            fields: ['size'],
+            value: 'Size',
+          },
+        ],
+      },
+    },
+    ['prefix', 'theme', 'size', 'dropdown']
+  );
+
+  const handleItemClick = (index: number) => {
+    setSelectedItem(index);
+    dropdown.selectedItem = dropdown?.items[index]?.label;
+  };
+
+  const handleTab =
+    (index: number) => (event: React.KeyboardEvent<HTMLElement>) => {
+      switch (event.key) {
+        case 'Tab':
+          event.preventDefault();
+          setSelectedItem(index);
+          dropdown.selectedItem = dropdown?.items[index]?.label;
+          setShow(!show);
+          break;
+        default:
+          break;
+      }
+    };
+
+  const toggleShow = () => {
+    setShow(!show);
+  };
+
+  return (
+    <>
+      <ThemeProvider
+        prefixes={{
+          dropdown: 'owlui-dropdown',
+          btn: 'owlui-btn',
+          'dropdown-toggle': 'owlui-dropdown-toggle',
+          'dropdown-menu': 'owlui-dropdown-menu',
+          'dropdown-item': 'owlui-dropdown-item',
+        }}
+      >
+        <div className={styleMod.dropdownContainer}>
+          <h2>{dropdown?.label}</h2>
```

Comment:
You might not want to render an `h2` to every dropdown so we could replace it with a `<p>` and render it conditionally so it will be displayed only when the dev passes in the `label` prop.